Documentation/ReleaseNotes/3.18.md

```diff
@@ -60,7 +60,7 @@ New Examples in this Release
 New Utilities in this Release
 -----------------------------
 
-    Utilities/itkExtHdrs/fdstream.hpp
+    Utilities/itkExtHdrs/fdstream.hxx
```

Comment:
I think we should leave these old release notes unchanged.

Reply:
OK.